app/models/data_center_stat.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: data_center_stats
+#
+#  id                 :bigint           not null, primary key
+#  gossip_nodes_count :integer
+#  network            :string(191)
+#  validators_count   :integer
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  data_center_id     :bigint           not null
+#
+# Indexes
+#
+#  index_data_center_stats_on_data_center_id              (data_center_id)
+#  index_data_center_stats_on_network_and_data_center_id  (network,data_center_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (data_center_id => data_centers.id)
+#
+class DataCenterStat < ApplicationRecord
```

Comment:
Perhaps we should check uniqueness of the stat in the scope of data_center and network?